### PR TITLE
Adapt to the renaming of zeq to ZeroEQ

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by rdumusc** [comment]
Blocked because of no tests on Ubuntu I think

File: apps/monsteerGUI/CMakeLists.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # This file is part of Monsteer <https://github.com/BlueBrain/Monsteer>
 #
 
-if(NOT ZEQ_FOUND)
+if(NOT ZEROEQ_FOUND)
```
Comment (tribal-tec) [inline apps/monsteerGUI/CMakeLists.txt]: if(NOT TARGET ZeroEQHBP)

**Comment by tribal-tec** [comment]
+1
Ubuntu Jenkins fix wrt tests target is opened for review. You're assigned too :)

File: apps/monsteerGUI/CMakeLists.txt (hunk 1 above)
Comment (rdumusc) [inline apps/monsteerGUI/CMakeLists.txt]: done

**Comment by rdumusc** [comment]
retest this please

**Pull request merged**